src/code/z_vismono.c

```diff
@@ -1,14 +1,181 @@
+/*
+ * File: z_vismono.c
+ * Description: Color frame buffer effect to desaturate the colors.
```

Comment:
```suggestion
 * @file z_vismono.c
 * @brief Color frame buffer effect to desaturate the colors.
```
For the sake of Doxygen.

Reply:
Is this something we want to be doing more broadly? As of now, I see only three files in the entire codebase use @file: sys_matrix.c, audio_load.c, and z_en_holl.c. The rest of the codebase uses the pattern I use here.

Reply:
I was planning to gradually switch over as we Doxygenify the codebase rather than try to get it all at once: it's a good indicator of stuff that will need to be adapted. That said, there's a few things in here that would also need adjustment, so you can leave it for now.

Reply:
Sure, I'll leave it for now.